src/asmcnc/skavaUI/screen_stop_or_resume_decision.py

```diff
@@ -174,7 +174,7 @@ def popup_help(self):
                 main_string=info,
                 popup_type=PopupType.QR,
                 popup_image=self.qr_source,
-                popup_image_size_hint=(1, 1),
+                popup_image_size_hint=(1.5, 1),
```

Comment:
You've said

> when I used 1.5, it moved the image to the right side of the popup, didn't look right

But then added it back in? I'm confused. How does it look now?